### DISCUSSION TITLE
casual/2024/03/03/%E5%88%B0%E8%AF%BE%E7%8E%87%E5%A4%A7%E4%B8%80%E4%B8%8B/

### DISCUSSION THREAD

**Discussion by giscus** [shrike-505/Nest#8] [Announcements]
# casual/2024/03/03/%E5%88%B0%E8%AF%BE%E7%8E%87%E5%A4%A7%E4%B8%80%E4%B8%8B/

https://shrike505.cc/casual/2024/03/03/%E5%88%B0%E8%AF%BE%E7%8E%87%E5%A4%A7%E4%B8%80%E4%B8%8B/

**Comment by Demetrius-Marlow** [comment]
兄弟啊😭

**Comment by zjyinzju** [comment]
一直翘课一直爽

**Comment by AphrixZjr** [comment]
迟到的）完结撒花！
